tests/test_discovery.py

```diff
@@ -7,49 +6,36 @@
 class CloseioDiscoveryTest(DiscoveryTest, CloseioBase):
     """Standard Discovery Test"""
 
+    expected_replication_keys = CloseioBase.expected_replication_keys
```

Comment:
This is here because pylint didn't know that we were inheriting from CloseioBase.  This is just so that pylint was happy, but functionally this doesn't do anything because the inheritance does the same thing.